src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -63,6 +86,12 @@ export default Utils.withCancellationSignal(class LaunchAnalysisModal extends Co
       ]),
       div({ style: { color: colors.danger(), overflowWrap: 'break-word' } }, [
         h(Fragment, wrappableOnPeriods(launchError))
+      ]),
+      location && locationType && span({}, [
+        'Data produced by this run will be output to the following workspace bucket',
```

Comment:
I still believe we should reword this to make the promise less specific and drop the disclaimer.

```suggestion
        'Data written to the workspace bucket resides in region `region-name`'
```

Reply:
I'm reluctant to switch to this wording.

1. I think 'Data written to the workspace bucket resides in region X' ignores the most important part of the story w.r.t. submitting a workflow - namely that this workflow will do some writing of data to that bucket.
1. I think "your outputs will be saved to X region" is true, even if it's not the full story.
1. I prefer optimizing the message for the common case and providing disclaimers for power users rather than making the message less useful in order to cover edges cases which the 80% will never have to worry about.

I'll also make the weak-sauce "resort to prior art" argument that the "Outputs" tab already has a form of words which is essentially identical to the statement being made here.